.eslintrc.json

```diff
@@ -11,27 +12,71 @@
 		"browser": true,
 		"node": true
 	},
+	"globals": { "Proxy": "readonly" },
+	"extends": [
+		"eslint:recommended",
+		"plugin:@typescript-eslint/recommended",
+		"plugin:@stylistic/disable-legacy"
+	],
+	"plugins": ["@typescript-eslint", "@stylistic"],
 	"rules": {
-		"semi": 1,
+		"@stylistic/arrow-spacing": 1,
+		"@stylistic/brace-style": [1, "stroustrup"],
+		"@stylistic/comma-spacing": 1,
+		"@stylistic/eol-last": 1,
+		"@stylistic/indent": [1, "tab"],
+		"@stylistic/keyword-spacing": 1,
+		"@stylistic/no-mixed-spaces-and-tabs": [1, "smart-tabs"],
+		"@stylistic/no-trailing-spaces": 1,
+		"@stylistic/quotes": [
+			1,
+			"double",
+			{ "avoidEscape": true, "allowTemplateLiterals": true }
+		],
+		"@stylistic/semi": 1,
+		"@stylistic/space-before-function-paren": 1,
+		"@stylistic/spaced-comment": [
+			1,
+			"always",
+			{ "block": { "exceptions": ["*"] } }
+		],
+		"@typescript-eslint/no-explicit-any": 0,
+		"@typescript-eslint/no-loss-of-precision": 1,
+		"@typescript-eslint/no-unused-vars": [1, { "args": "none" }],
+		"curly": 1,
 		"no-dupe-args": 1,
 		"no-dupe-keys": 1,
+		"no-empty": 0,
 		"no-unreachable": 1,
-		"valid-typeof": 1,
-		"curly": 1,
 		"no-useless-call": 1,
-		"brace-style": [1, "stroustrup"],
-		"no-mixed-spaces-and-tabs": [1, "smart-tabs"],
-		"quotes": [1, "double", {"avoidEscape": true, "allowTemplateLiterals": true}],
-		"spaced-comment": [1, "always", {
-			"block": {
-				"exceptions": ["*"]
+		"valid-typeof": 1
+	},
+	"overrides": [
+		{
+			"files": [
+				"apps/**/*",
+				"assets/**/*",
+				"get/**/*",
+				"notebook/**/*",
+				"scripts/**/*",
+				"tests/**/*"
+			],
+			"rules": {
+				"@typescript-eslint/no-unused-vars": 0,
```

Comment:
Not sure I want to enforce this, sometimes having unused vars can be useful for documentation purposes or to iterate.